README.md

```diff
@@ -111,123 +72,84 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Comming Soon 🎈]()
+- Please I have issues deploying the app on render on several attempts. I am still working to resolve that.
```

Comment:
- [ ] Hi, please note that deployment is a core requirement for this project and cannot be approved without it. I would advise you to resolve the issue before requesting another review.